deployer/hub.py

```diff
@@ -458,7 +458,20 @@ def deploy(self, auth_provider, secret_key, skip_hub_health_test=False):
                 # This can contain sensitive info - so we hide stderr
                 # FIXME: Don't use pytest - just call a function instead
                 print("Running hub health check...")
-                with open(os.devnull, 'w') as dn, redirect_stderr(dn), redirect_stdout(dn):
+                # Show errors locally but redirect on CI
+                gh_ci = os.environ.get('CI', "false")
+                if gh_ci == "true":
+                    print("Testing on CI, redirected output")
```

Comment:
```suggestion
                    print("Testing on CI, not printing output")
```
just to be a bit more explicit